config/locales/views/company.sv.yml

```diff
@@ -0,0 +1,42 @@
+sv:
+  company:
+    about:
```

Comment:
This is the kind of material you do not want to hardcode into your app... Becuause if you do that means you are responsible for maintaining it.